code/modules/surgery/organs/stomach.dm

```diff
@@ -251,7 +251,7 @@
 		var/message = input(owner, "What message would you like to imprint on the deposit?", "Deposit message:") as text|null
 		if(!message)
 			return
-		owner.log_game("[key_name(owner)] as [owner] made a new spore deposit with the message '[message]'.", type=LOG_GAME)
+		log_game("[key_name(owner)] as [owner] made a new spore deposit with the message '[message]'.", type=LOG_GAME)
```

Comment:
I did say log_message

Reply:
`owner.log_message("made a new spore deposit with the message '[message]'", type=LOG_GAME)`